src/OrbitGl/ThreadTrack.cpp

```diff
@@ -69,22 +68,27 @@ std::string ThreadTrack::GetThreadNameFromTid(uint32_t thread_id) {
   ;
 }
 
-void ThreadTrack::InitializeNameAndLabel() {
+const std::string_view kAllThreads = " (all_threads)";
+const std::string ThreadTrack::GetLabel() const {
+  auto thread_id = GetThreadId();
+  auto name = GetName();
+  if (thread_id == orbit_base::kAllThreadsOfAllProcessesTid) {
+    return name;
+  } else if (thread_id == orbit_base::kAllProcessThreadsTid) {
+    return name.append(kAllThreads);
+  }
+  std::string tid_str = std::to_string(thread_id);
+  return absl::StrFormat("%s [%s]", name, tid_str);
```

Comment:
`thread_id` is a number, right? `StrFormat` could do the formatting, so no call to `to_string` is needed, right?